tests/sys/kern/module_test.c

```diff
@@ -0,0 +1,193 @@
+/*-
+ * Copyright (c) 2022 XXX
```

Comment:
:eyes:

Reply:
Yeah, waiting to here from @rwatson what to put here.  The tests I can also upstream directly to FreeBSD.